cfpb/cfgov-refresh
# Update post previews in filterable lists

This is in response to stakeholder feedback in the post previews.

## Additions

- `post_date_description` in FilterControls organism
- `show_preview_categories` in FilterControls organism

## Testing

- Go to a filterable list page
- Categories should be seen on the post previews and the date should say `Published <date>`
- Toggle the categories in the FilterControls
- Change the date description
 - Changes should be shown in the list

## Review

- @richaagarwal 
- @kave 
- @rosskarchner
